src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
 
+    static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
What about access modifiers?

Reply:
I use FORMAT only in this class, so I changed it to private.